docs/en-US/component/cascader.md

```diff
@@ -90,7 +90,7 @@ cascader/multiple-selection
 
 In single selection, only the leaf nodes can be checked, and in multiple selection, check parent nodes will lead to leaf nodes be checked eventually. When enable this feature, it can make parent and child nodes unlinked and you can select any level of options.
 
-:::demo Set `props.checkStrictly = true` to make checked state of a node not affects its parent nodes and child nodes, and then you can select any level of options.
+:::demo Set `props.check-strictly = true` to make checked state of a node not affects its parent nodes and child nodes, and then you can select any level of options.
```

Comment:
Same here.

Reply:
but check-strictly in other file such as TreeSelect and Tree

Reply:
`props.check-strictly` is illegal, kebab-case should only use in HTML attribution

Reply:
got it